types/bigtable.go

```diff
@@ -18,14 +18,16 @@ type ValidatorBalanceStatistic struct {
 }
 
 type ValidatorMissedAttestationsStatistic struct {
```

Comment:
**Minor:** Could also be renamed to ValidatorFailedAttestationsStatistic

Reply:
sure, renamed it